tests/virt_autotest/login_console.pm

```diff
@@ -35,6 +35,7 @@ sub login_to_console() {
     assert_screen([qw(grub2 grub1)], 120);
     if (!get_var("reboot_for_upgrade_step")) {
         if (get_var("XEN") || check_var("HOST_HYPERVISOR", "xen")) {
+            send_key 'up';
```

Comment:
To the comment for line 38 of send_key up, I will add comment. And of course we do a lot of special handlings regarding different virtualization scenarios in this file comparing with grub_test.pm.